### PR TITLE
Fixed linter error in hpp file.

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
Signed-off-by: Ben Starmer-Smith <ben@shadowrobot.com>

## Proposed changes

Fixed hpp linter error.

## Checklist

If the task is applicable to this pull request (see applicability criteria in brackets), make sure it is completed before checking the corresponding box. Otherwise, tick the box right away. Make sure that **ALL** boxes are checked **BEFORE** the PR is merged.

- [ ] Manually tested that added code works as intended (if any functional/runnable code is added).
- [ ] Added automated tests (if a new class is added (Python or C++), interface of that class must be unit tested).
- [ ] Tested on real hardware (if the changed or added code is used with the real hardware).
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc.)

### CODE REVIEW

**Review by mykolasjuraitis — APPROVED** [review]

File: sr_robot_lib/include/sr_robot_lib/MST.hpp [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 #include <vector>
 #include <string>
+#include <memory>
```
Comment (carebare47) [inline sr_robot_lib/include/sr_robot_lib/MST.hpp:24]: Do you have the error output, or a way to run the linter locally? What was the complaint from the linter that resulted in this?

Comment (mykolasjuraitis) [reply sr_robot_lib/include/sr_robot_lib/MST.hpp:24]: It must be "Include what you use" error for `std::shared_ptr`.

Comment (BenStarmerSmith) [reply sr_robot_lib/include/sr_robot_lib/MST.hpp:24]: I can run it locally, but this is the error given by the linter: 
`Error 2 in /root/workspace/src/codebuild/output/src417757905/src/github.com/shadow-robot/sr-ros-interface-ethercat/sr_robot_lib/include/sr_robot_lib/MST.hpp:66:  Add #include <memory> for shared_ptr<>  [build/include_what_you_use]`

Comment (carebare47) [reply sr_robot_lib/include/sr_robot_lib/MST.hpp:24]: Ahh ok, thanks all

**Review by carebare47 — APPROVED** [review]

**Review by georgiablanco — APPROVED** [review]

**Pull request merged**